includes/polling/os/hpe-msa.inc.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+$oids = 'cpqSiProductName.0 cpqSiSysSerialNum.0';
+
+$data = snmp_get_multi($device, $oids, '-OQUs', 'CPQSINFO-MIB');
+
+$hardware = preg_replace('/"/', '', $data[0]['cpqSiProductName']);
```

Comment:
What is the output for these?  I don't like the preg_replace here when `str_replace()` or `trim($sr, '"')` would likely work.

Adding the output of these to the snmprec wouldn't be a bad idea either. like this:
```
1.3.6.1.4.1.232.2.2.4.2.0|4|Product name string here
1.3.6.1.4.1.232.2.2.2.1.0|4|Serial number here (Change actual number to a similar format/length)
```